src/renderer/angular/directives/commons/sk-tx-history.directive.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
+/* eslint-disable */
```

Comment:
why disable enlist ?

Reply:
@sk91 sorry about this, the line is removed now.